Aztec/Classes/Public/Extensions/CGRect+Helpers.swift

```diff
@@ -0,0 +1,56 @@
+import CoreGraphics
+
+extension CGRect {
+    /// The position to which a rect is anchored when resized vertically.
+    ///
+    enum VerticalAnchor {
+        case top
+        case middle
+        case bottom
+    }
+
+    /// The position to which a rect is anchored when resized horizontally.
+    ///
+    enum HorizontalAnchor {
+        case left
+        case center
+        case right
+    }
+
+    /// Return a rect resized to a specific size, while anchoring it to one of
+    /// their edges or centers.
+    ///
+    /// By default, changing a CGRect size will resize it anchored to its top
```

Comment:
> This support is offered in a really broad and generic way, non specific to UIKit

I agree that the explanation is perhaps too detailed, but this is not generic as long as the horizontal/vertical enum values mention left/right and top/bottom. CoreGraphics actually has the origin at the bottom left corner, so if someone used this method within that context, the vertical enum values would mean the opposite of what they are.

I think it's safe to assume that will not happen within the scope of this project, but that's why I added the note about UIKit. If you think it adds more confusion I can remove it.